Add i18n instructions for English PRs to README (#732)
withastro/docs: README.md

This will automatically create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of the docs in your GitHub account with the changes.

Once your edits are ready in GitHub, follow the prompts to **create a pull request** and submit your changes for review.

> Every pull request needs to be reviewed by our contributors and approved by a Maintainer.

Reviewer: Is the capitalisation difference intentional here because of governance roles or something? Contributor and maintainer are both defined there right? I’d personally just go with lowercase here, but am not tied to that.

```suggestion
Every pull request needs to be reviewed by our contributors and approved by a maintainer.
```

**Important Note re: Internationalization (i18n)**

Please only add new text content to the docs **in English**, only modifying **`.md` files located within `src/pages/en/`**. 

We have automated systems in place for notifying our community translators that there is new material to be translated, so there is no need to make changes to additional languages yourself. 

> _Speak another language natively? Join our i18n gang on Discord or jump into the PRs to help with translating or reviewing translations!_ 

Reviewer: (GitHub wouldn’t let me create a suggestion for this one sorry)

> _Speak another language natively? Join our i18n gang on Discord or jump into the PRs to help with translating or reviewing translations!_ 

Reviewer: Just this outstanding comment left — but otherwise LGTM. Thanks for this update Sarah!

> _Speak another language natively? Join our i18n gang on Discord or jump into the PRs to help with translating or reviewing translations!_ 

Author: Oops, missed this one! Thanks!

> _Speak another language natively? Join our i18n gang on Discord or jump into the PRs to help with translating or reviewing translations!_ 

Author: ```suggestion

```

## Translate a Page

> 

Author: ```suggestion
Speak another language natively? Join our i18n gang on Discord or jump into the PRs to help with translating or reviewing translations!
```

We are not strict about branch names.

```shell
git checkout -b add/partial-hydration-typo-fix
```

That’s it.

> As you [open a pull request](https://github.com/withastro/astro/compare), please include a clear title and description.

Author: ```suggestion
As you [open a pull request](https://github.com/withastro/astro/compare), please include a clear title. The description will be pre-filled with questions that you can answer by editing right in the text field.
```

```markdown
# Fixes a typo on Partial Hydration page

This fixes a typo in the page on partial hydration, and additionally removes an unnecessary extra space.

> ```

Reviewer: Maybe we can get away without this example here now that PRs are pre-populated with a template?

Author: I like it even better!
